cristin-project/src/main/java/no/unit/nva/cristin/projects/CreateCristinProjectHandler.java

```diff
@@ -67,37 +68,33 @@ protected NvaProject processInput(NvaProject input, RequestInfo requestInfo, Con
     }
 
     private void validateInput(NvaProject project) throws BadRequestException {
-        if (hasId(project)
-                || noTitle(project)
-                || invalidStatus(project)
```

Comment:
Removed status field check as it is not required in Cristin. Also removed it from karate